workers/social/lib/social/models/user/index.coffee

```diff
@@ -643,6 +637,13 @@ module.exports = class JUser extends jraphical.Module
       return callback null, { isEligible: yes }
 
 
+  logout = (reason, clientId, callback) ->
```

Comment:
could we change the name to a more specific one like "terminateSessoin" when i first read logout, i thought you forget error handling in the function and outputting errors but turned out it is only called on error cases.